common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -307,7 +308,7 @@ if Backbone?
         model: @model
         mode: @mode
         context: @context
-        course_settings: @options.course_settings
+        course_settings: @course_settings
```

Comment:
In short: Backbone Views no longer automatically attach options passed to the constructor as this.options and Backbone Models no longer attach url and urlRoot options.